### DISCUSSION TITLE
Failure oracle-libs installation

### DISCUSSION THREAD

**Discussion by ec-aoki** [docker-library/postgres#1231] [General]
# Summary
Since last week, I have been experiencing failures during the installation of the oracle-libs package. I have found a temporary workaround, but I am unsure where to report this issue.

# Expected behaviour
Successful installation of the oracle-libs package.

```
docker run --rm -it postgres:15.4-bookworm bash -c 'apt update && apt install -y oracle-libs'
```

# Current behaviour
Failure during the installation of the oracle-libs package.

```
docker run --rm -it postgres:15.4-bookworm bash -c 'apt update && apt install -y oracle-libs'
```

# Steps to reproduce
After examining the logs from the failed installation, I found that dpkg-architecture was required. Adding the dpkg-dev package resolved the issue, and the installation succeeded.

```
docker run --rm -it postgres:15.4-bookworm bash -c 'apt update && apt install -y dpkg-dev && apt install -y oracle-libs'
```

An excerpt from the logs I referenced:
In the shell script /usr/sbin/oracle-libs-update, an attempt to use dpkg-architecture failed.

```
/usr/sbin/oracle-libs-update: 39: dpkg-architecture: not found
dpkg: error processing package oracle-libs (--configure):
installed oracle-libs package post-installation script subprocess returned error exit status 127
```

# Additional context

I confirmed that there was an update to the oracle-libs package last week.

https://www.postgresql.org/message-id/E1rweXv-0035w1-Gr%40atalia.postgresql.org

**Comment by tianon** [comment]
Unfortunately, we do not install, maintain, or support the `oracle-libs` package you're installing.  It looks like the update probably missed a dependency on `dpkg-dev` (https://packages.debian.org/file:dpkg-architecture).

**Reply by ec-aoki** [reply]
Thank you. I'll reach out to the team maintaining the package.
